rclcpp/include/rclcpp/executor.hpp

```diff
@@ -329,6 +448,15 @@ class Executor
   rclcpp::node_interfaces::NodeBaseInterface::SharedPtr
   get_node_by_group(rclcpp::CallbackGroup::SharedPtr group);
 
+  /**
+   * Checks in `weak_groups_to_nodes` whether `node_ptr` has been added to the
```

Comment:
Use a brief with `///`, also avoid discussing internals of the class, and instead just describe what should happen.